dds/DCPS/GuidConverter.cpp

```diff
@@ -206,6 +206,14 @@ GuidConverter::uniqueId() const
   return id;
 }
 
+#ifdef OPENDDS_SECURITY
+bool
+GuidConverter::isSecure() const
```

Comment:
Should this also check that entityKind is a built-in kind?  Otherwise we don't know how entityKey[0] is assigned.